packages/app-extensions/src/formField/formField.js

```diff
@@ -30,15 +30,17 @@ export const formFieldFactory = (mapping, data, resources = {}) => {
     const mandatory = !readOnly && _get(modelField, `validation.mandatory`, false)
 
     return (
-      <FormField
-        key={id}
+      <StatedValue
```

Comment:
If we have a readOnly form (a form where root tag has readOnly=true attribute) we show all fields as FormattedValues instead of EditableValues. Have you considered this case?

With mocked data this can be tested by opening a record of the subtable in the detail.